src/Trixi.jl

```diff
@@ -161,8 +165,9 @@ export initial_condition_monopole, boundary_condition_monopole
 
 export cons2cons, cons2prim, prim2cons, cons2macroscopic, cons2state, cons2mean,
        cons2entropy, entropy2cons
-export density, pressure, density_pressure, velocity
+export density, pressure, density_pressure, velocity, waterheight, waterheight_pressure
```

Comment:
Do you use `waterheight_pressure` somewhere? If not, I think we should remove it

Reply:
It is analogous routine to the `density_pressure` from compressible Euler. It is not used in this PR but is probably needed in shock capturing. Should I just save it locally for now?

Reply:
I would say we can add it when it's needed. There is no need to commit stuff that is not used at all at the moment.

Reply:
I removed the functions `pressure` and `waterheight_pressure` from the shallow water equations file as we will not need them until #736 is addressed. I also no longer export `waterheight` because it is only needed internally in shallow water for the time being.